src/main.c

```diff
@@ -1,14 +1,84 @@
-#include "uart.h"
+#include "board_definitions.h"
+#include "hardware_functions.h"
+#include "error_handling.h"
+#include "calipile_discovery.h"
+#include "VL53L1_platform.h"
+#include "VL53L1X_api.h"
+#include "VL53L1X_calibration.h"
+
+
+
+#define CALIPILE_0_ADDR (0x0C<<1)
+#define CALIPILE_1_ADDR (0x0D<<1)
+
+Calipile sensor0 = {CALIPILE_0_ADDR,11,8,10,50,10,10,0x00,0x08,0x01,0x00,0,0};
```

Comment:
zit er logica in deze magie? kan er iets met een define worden gemaakt?

Reply:
snap er indd geen fuck van, mag je best weten @joostje95

Reply:
kusje